src/Core/Managed/Shared/Extensibility/Implementation/Tracing/CoreEventSource.cs

```diff
@@ -399,6 +399,26 @@ public void FailedToLoadEnvironmentVariables(string ex, string appDomainName = "
             this.WriteEvent(32, ex, this.nameProvider.Name);
         }
 
+        // Verbosity is Error - so it is always sent to portal; Keyword is Diagnostics so throttling is not applied.
+        [Event(33,
+            Message = "A Metric Extractor detected a telemetry item with SamplingPercentage < 100. Metrics Extractors should be used before Sampling Processors or any other Telemetry Processors that might filter out Telemetry Items. Otherwise, extracted metrics may be incorrect.",
```

Comment:
I meant to pass an actual sampling percentage and item type we found instead of ` SamplingPercentage < 100`. Consider updating, but not a big deal.